app/web/features/profile/locales/en.json

```
@@ -127,37 +128,61 @@
   "report_reason": "Reason",
   "report_user": "Report this user",
   "leave_reference": {
+    "add_a_reference": "Add a reference",
```

We should be a bit more clear about where/how the reference is shared with the community:

```suggestion
    "add_a_reference": "Add a reference to {{name}}'s profile",
```